app/static/app.js

```diff
@@ -394,25 +395,39 @@ function open_tap_error() {
 }
 
 const tap_source = new EventSource("/api/tap-source");
-tap_source.onmessage = function () {
+
+tap_source.onmessage = function (e) {
```

Comment:
```suggestion
tap_source.onmessage = function (message) {
```

Single character variables are discouraged... not sure how the linter didn't catch this one.

So maybe call it `message` or `event`.

Reply:
Done.